zboxcore/sdk/blockdownloadworker.go

```diff
@@ -204,6 +204,7 @@ func (req *BlockDownloadRequest) downloadBlobberBlock() {
 				if bytes.Contains(respBody, []byte(LockExists)) {
 					zlogger.Logger.Debug("Lock exists error.")
 					shouldRetry = true
+					time.Sleep(time.Second * 1)
```

Comment:
pleas use sys.Sleep, if not, main thread will be blocked in wasm sdk.

Reply:
@cnlangzi if the time.Sleep will cause problem, please fix this in a new PR, thanks.

Reply:
@peterlimg it is fixed